src/components/transactions/ExecuteTxButton/index.tsx

```diff
@@ -26,9 +27,19 @@ const ExecuteTxButton = ({
   const txNonce = isMultisigExecutionInfo(txSummary.executionInfo) ? txSummary.executionInfo.nonce : undefined
   const isPending = useIsPending(txSummary.id)
   const { setSelectedTxId } = useContext(ReplaceTxHoverContext)
+  const safeSDK = getSafeSDK()
 
   const isNext = txNonce !== undefined && txNonce === safe.nonce
-  const isDisabled = !isNext || isPending
+  const isDisabled = !isNext || isPending || !safeSDK
+
+  const tooltipTitle =
+    isDisabled && !isNext
+      ? `Transaction ${safe.nonce} must be executed first`
+      : isPending
+      ? `There's a pending transaction`
+      : !safeSDK
+      ? 'Waiting for the SDK to initialize'
```

Comment:
I'm not sure about this - it's somewhat technical for the "standard" user. It shouldn't happen to too long either.

Reply:
I've answered in https://github.com/safe-global/safe-wallet-web/pull/1969/files#r1189586069